client/src/components/SoundPanel.tsx

```diff
@@ -73,6 +74,12 @@ interface ISoundPanelProps {
 	sendSound: (soundText: string, soundType: string) => void;
 }
 
+interface ICategoryProps {
+	category: SoundCategories;
+	activeCategory: SoundCategories;
```

Comment:
Another instance of coupling we want to avoid. Category doesn't need to know what the active category is, it'd be more appropriate to have a boolean prop like isActive, which is set through the parent component. Ideally, components should only know what they need to know, should only be responsible for one thing